### ISSUE TITLE
Had to enable transpiling for Jest tests to run

### ISSUE THREAD

**Issue by phillipj** [expo/sentry-expo#12]
Hi,

thanks so much for streamlining Sentry implementation, really smooth getting everything up-n-running 👍 Thought I share my fix getting over a small hurdle for tests to run.

I had some issues running the tests of a newly created CRNA project:

```bash
$ npm test

..

 FAIL  ./App.test.js
  ● Test suite failed to run

    /.../react-native-ui/node_modules/sentry-expo/index.js:3
    import { Constants } from 'expo';
    ^^^^^^

    SyntaxError: Unexpected token import

      at ScriptTransformer._transformAndBuildScript (node_modules/jest-runtime/build/ScriptTransformer.js:289:17)
      at Object.<anonymous> (App.js:3:17)
      at Object.<anonymous> (App.test.js:2:10)

Test Suites: 1 failed, 1 total
Tests:       0 total
Snapshots:   0 total
Time:        0.348s, estimated 1s
Ran all test suites.
npm ERR! Test failed.  See above for more details.
```

After some digging, I found that I had to enable transpiling of contents in `node_modules` since this package is published with the use ES-modules. Luckily Jest is easily configured by reseting the [transformIgnorePatterns](https://facebook.github.io/jest/docs/configuration.html#transformignorepatterns-array-string) field in `package.json`:

```diff
   "jest": {
-    "preset": "jest-expo"
+    "preset": "jest-expo",
+    "transformIgnorePatterns": []
   },
```

This seems to increase the time used by Jest to run/build the tests slightly, though I've only tested this on a clean CRNA project.

Have you considered transpiling this package, or using `require()` for the time being until Node.js gets built-in ES-modules support?

**Comment by JeremyLoy** [comment]
Thanks @phillipj for the quick fix.

This bug added significantly to my test durations. Anyone had a chance to look at it yet?

**Comment by burkeshartsis** [comment]
### TLDR;
Update `jest-expo` and it should fix this problem as `sentry-expo` is included in it's [preset `transformIgnorePatterns`](https://github.com/expo/jest-expo/blob/0dec72042e8dade35dc6da26c1dec1bcbf7a37eb/jest-preset.json#L27)

### If you need custom `transformIgnorePatterns`
I ran into this problem as well. Resetting `transformIgnorePatterns` isn't ideal and in some instances could cause issues with other packages. Copying your setups patterns and appending `sentry-expo` is probably what you really want to do. The default as per [Jest docs](https://facebook.github.io/jest/docs/en/configuration.html#transformignorepatterns-array-string) is
```
["/node_modules/"]
```
If you're using CRNA it might look something like
```
["node_modules/(?!((jest-)?react-native|react-clone-referenced-element|expo(nent)?|@expo(nent)?/.*|react-navigation))"]
```
For additional reading see [https://facebook.github.io/jest/docs/en/tutorial-react-native.html#transformignorepatterns-customization](https://facebook.github.io/jest/docs/en/tutorial-react-native.html#transformignorepatterns-customization)

**Comment by phillipj** [comment]
That sounds awesome, thanks for the heads up! I haven't tested it myself, but I trust you and close this issue for now.
